addons/web_editor/security/ir.model.access.csv

```diff
@@ -1,3 +1,7 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_web_editor_converter_test,access_web_editor_converter_test,model_web_editor_converter_test,base.group_system,1,1,1,1
 access_web_editor_converter_test_sub,access_web_editor_converter_test_sub,model_web_editor_converter_test_sub,base.group_system,1,1,1,1
+access_web_editor_field_history_all,access.web.editor.field.history.all,model_field_html_history_revision,,0,0,0,0
+access_web_editor_field_history_portal,access.web.editor.field.history.portal,model_field_html_history_revision,base.group_portal,1,0,1,0
```

Comment:
Not sure if you are still WIP or not (seems PR is not in draft anyway) this looks like a security issue to me.

Reply:
It is not in draft anymore.
Could you explain why this seems like a security issue ? it seems ok to me, but I'm not at all confident to write access rights.

Reply:
You give create and read permissions to portal without any linked rule. It means he can do whatever he wants with history, which is probably not what you want. Using a script a portal user can create diff in about all models using your mixin, leading to a lot of potential issues :) .

I think portal should not be allowed to do anything with history, using sudo() to create the revisions when the write access is already granted on the parent record for example.

Reply:
OK. So If I understand well you recommend removing access right to most user and just use `sudo()` in the python model to create the revision records.

Will check this.